### PR TITLE
Feature/e UI 3887 tasks api

### PR DESCRIPTION
**Pull request by kuda1992**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-3887
https://tools.hmcts.net/jira/browse/EUI-3571

### Change description ###
Add a getTasksByCaseId mock api
Add a resolve to get tasks by case id 
Get the tasks in the TasksContainerComponent from ActivatedRoute
Add TaskAlertBanner when warning is present in one of the obtained tasks

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x ] No
```

### CODE REVIEW

**Review by udaydenduluri33 — APPROVED** [review]
Approved @connorpgpmcelroy  @kuda1992  
Both Mock and UI changes

File: src/cases/components/task-alert-banner/task-alert-banner.component.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'exui-task-alert-banner',
+  templateUrl: './task-alert-banner.component.html',
+  styleUrls: ['./task-alert-banner.component.scss']
+})
+export class TaskAlertBannerComponent {
+  public alertTitle = 'Task alert';
+  public alertMessage = 'An application is pending a decision. Consider if this application has an impact on any other tasks you might wish to complete.';
```
Comment (rayliangatos) [inline src/cases/components/task-alert-banner/task-alert-banner.component.ts:10]:
```suggestion
  @Input() public alertTitle;
  @Input() public alertMessage;
```
If this is component, it should be a generic component and passed from the container.

File: src/cases/containers/tasks-container/tasks-container.component.html [hunk 2]
```diff
@@ -1,4 +1,5 @@
 <exui-alert></exui-alert>
+<exui-task-alert-banner *ngIf="warningIncluded"></exui-task-alert-banner>
```
Comment (rayliangatos) [inline src/cases/containers/tasks-container/tasks-container.component.html:2]:
```suggestion
<exui-task-alert-banner *ngIf="warningIncluded"
[alertTitle]="Task alert"
[alertMessage]="An application is pending a decision. Consider if this application has an impact on any other tasks you might wish to complete."
></exui-task-alert-banner>
```

File: src/cases/components/task-alert-banner/task-alert-banner.component.ts (hunk 1 above)
Comment (connorpgpmcelroy) [reply src/cases/components/task-alert-banner/task-alert-banner.component.ts:10]: Yes, that is a good point. Will change

File: src/cases/containers/tasks-container/tasks-container.component.html (hunk 2 above)
Comment (connorpgpmcelroy) [reply src/cases/containers/tasks-container/tasks-container.component.html:2]: Similar to above - agreed

**Review by rayliangatos — APPROVED** [review]

**Review by sreekanthpuligadda — APPROVED** [review]
reviewed test coverage, looks good

**Pull request merged**